build-debezium.sh

```diff
@@ -24,47 +42,40 @@ build_docker_image () {
-    echo "** Building    debezium/${IMAGE_NAME}:${IMAGE_TAG}"
+    echo "** Building    ${DEBEZIUM_DOCKER_NAME}/${IMAGE_NAME}:${IMAGE_TAG}"
     echo "****************************************************************"
-    docker build -t "debezium/${IMAGE_NAME}:latest" "${IMAGE_PATH}"
+
+    TAGS=("-t ${DEBEZIUM_DOCKER_NAME}/${IMAGE_NAME}:latest")
 
     if [ -z "$RELEASE_TAG" ]; then
         echo "****************************************************************"
-        echo "** Stream Tag  debezium/${IMAGE_NAME}:${IMAGE_TAG}       "
+        echo "** Stream Tag  ${DEBEZIUM_DOCKER_NAME}/${IMAGE_NAME}:${IMAGE_TAG}       "
         echo "****************************************************************"
-        docker tag "debezium/${IMAGE_NAME}:latest" "debezium/${IMAGE_NAME}:${IMAGE_TAG}"
-        if [ "$PUSH_IMAGES" == "true" ]; then
-            echo "Pushing the stream image into the registry"
-            docker push "debezium/${IMAGE_NAME}:${IMAGE_TAG}"
-            docker tag "debezium/${IMAGE_NAME}:${IMAGE_TAG}" "quay.io/debezium/${IMAGE_NAME}:${IMAGE_TAG}"
-            docker push "quay.io/debezium/${IMAGE_NAME}:${IMAGE_TAG}"
-            if [ "$DEBEZIUM_VERSION" == "$LATEST_STREAM" ]; then
-                echo "Pushing the latest image into the registry"
-                docker push "debezium/${IMAGE_NAME}:latest"
-                docker tag "debezium/${IMAGE_NAME}:latest" "quay.io/debezium/${IMAGE_NAME}:latest"
-                docker push "quay.io/debezium/${IMAGE_NAME}:latest"
-            fi
-        fi
-    fi
+        TAGS+=("-t ${DEBEZIUM_DOCKER_NAME}/${IMAGE_NAME}:${IMAGE_TAG}")
+        TAGS+=("-t localhost:5000/${DEBEZIUM_DOCKER_NAME}/${IMAGE_NAME}:${IMAGE_TAG}")
+    fi;
 
     if [ -n "$RELEASE_TAG" ]; then
         echo "****************************************************************"
-        echo "** Release Tag debezium/${IMAGE_NAME}:${RELEASE_TAG}       "
+        echo "** Release Tag ${DEBEZIUM_DOCKER_NAME}/${IMAGE_NAME}:${RELEASE_TAG}       "
         echo "****************************************************************"
-        docker tag "debezium/${IMAGE_NAME}:latest" "debezium/${IMAGE_NAME}:${RELEASE_TAG}"
-        if [ "$PUSH_IMAGES" == "true" ]; then
-            echo "Pushing the stream image into the registry"
-            docker tag "debezium/${IMAGE_NAME}:${RELEASE_TAG}" "quay.io/debezium/${IMAGE_NAME}:${RELEASE_TAG}"
-            docker push "quay.io/debezium/${IMAGE_NAME}:${RELEASE_TAG}"
-            docker push "debezium/${IMAGE_NAME}:${RELEASE_TAG}"
-        fi
+
+        TAGS+=("-t ${DEBEZIUM_DOCKER_NAME}/${IMAGE_NAME}:${RELEASE_TAG}")
+        TAGS+=("-t localhost:5000/${DEBEZIUM_DOCKER_NAME}/${IMAGE_NAME}:${RELEASE_TAG}")
     fi
+
+    echo "Build Image with Tags " "${TAGS[@]}"
+
+    docker buildx build --push --platform linux/amd64,linux/arm64 \
+      --build-arg DEBEZIUM_DOCKER_NAME="$DEBEZIUM_DOCKER_NAME" \
+        "${TAGS[@]}" \
```

Comment:
this set of quotes causes an issue where each `-t` element is actually passed as `'-t debezium/base:latest'` which is an incorrect expansion causing docker to complain about invalid reference format.

Reply:
Will fix that, thanks!